Doc/requirements.txt

```diff
@@ -3,11 +3,9 @@
 # Sphinx version is pinned so that new versions that introduce new warnings
 # won't suddenly cause build failures. Updating the version is fine as long
 # as no warnings are raised by doing so.
-sphinx==2.4.4
-# Docutils version is pinned to a version compatible with Sphinx
-# version 2.4.4. It can be removed after bumping Sphinx version to at
-# least 3.5.4.
-docutils==0.17.1
+sphinx==3.5.4
+# Jinja version is pinned to a version compatible with Sphinx version 3.5.4.
+jinja2==3.0.3
```

Comment:
Let's keep the Sphinx and Docutils pins and add a new Jinja2 pin:

```suggestion
sphinx==2.4.4
# Docutils version is pinned to a version compatible with Sphinx
# version 2.4.4. It can be removed after bumping Sphinx version to at
# least 3.5.4.
docutils==0.17.1
# Jinja version is pinned to a version compatible with Sphinx version 2.4.4.
jinja2==3.0.3
```

Reply:
I've updated the commit accordingly.